docs/content/en/contribute/general/github/_index.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Working with github
+description: Information relevant to github
+weight: 100
+---
+
+To start your contribution with
+[Keptn Lifecycle Toolkit](https://github.com/keptn/lifecycle-toolkit)
+follow this github instructions:
```

Comment:
Might want to say that all Keptn software and documentation is stored in github repositories and all changes are made using standard Git practices.  Most of the material in this section is just something to help people who are not Git experts but this is not an exhaustive list of what you can do with Git and, if you are an experienced Git user and have other practices that you prefer, you can use them.  My wording is pretty rough -- I'm sure you'll do better.